Goose.API/Repositories/UserRepository.cs

```diff
@@ -1,16 +1,19 @@
 ﻿using Goose.Data.Context;
 using Goose.Data.Repository;
-using Goose.Domain.Models.identity;
+using Goose.Domain.Models.Identity;
+using MongoDB.Driver;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
+#nullable enable
+
 namespace Goose.API.Repositories
 {
     public interface IUserRepository : IRepository<User>
     {
-
+        Task<User?> GetByUsernameAsync(string email);
```

Comment:
eine Kleinigkeit 
der Übergabeparameter heißt im Interface email und in der Implementierung username hatte mich etwas verwirrt 😅

Reply:
🤦‍♂️
2b64bd2